enr/enr.go

```diff
@@ -28,6 +31,40 @@ type ENR struct {
 	Udp6Port uint16   // IPv6-specific UDP port. If omitted, same as UdpPort.
 }
 
+// Returns the address of the node, which is the keccak256 hash of its Secp256k1 key.
+func (enr ENR) NodeAddr() []byte {
```

Comment:
I think this is the same as https://github.com/ethereum/go-ethereum/blob/4f7a425aa827f7c2d27b50385019b1602a3e1851/p2p/enode/idscheme.go#L81 but not entirely sure